sky/backends/backend_utils.py

```diff
@@ -1142,6 +1143,15 @@ def get_node_ips(cluster_yaml: str,
             handle.head_ip is not None):
         return [handle.head_ip]
 
+    # If get_internal_ips is True, we create a temporary config file
+    # with config[provider][use_internal_ips] set to True for
+    # "ray get-worker-ips"
```

Comment:
nit: this comment may not be necessary based on the system design principle: "Comments should not repeats code".

Reply:
agreed!